otter/rest/groups.py

```diff
@@ -160,14 +177,15 @@ def format_list(group_states):
         deferred.addCallback(json.dumps)
         return deferred
 
-    # --------------------------- CRD a scaling group -----------------------------
+    # -------------------------- CRD a scaling group -------------------------
     # (CRD = CRUD - U, because updating happens at suburls - so you can update
     # different parts)
 
-    # TODO: Currently, the response does not include scaling policy ids, because
-    # we are just repeating whatever the request body was, with an ID and links
-    # attached.  If we are going to create the scaling policies here too, we should
-    # probably also return their ids and links, just like the manifest.
+    # TODO: Currently, the response does not include scaling policy ids,
```

Comment:
Apologies, I know all you did was reflow this, but this TODO is no longer applicable (we return ids and links).  Can it be removed in this PR, or should there just be another PR?